cpp/include/Ice/TimeUtil.h

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#ifndef ICE_TIME_UTIL_H
+#define ICE_TIME_UTIL_H
+
+#include <Ice/Config.h>
+
+#include <chrono>
+#include <string>
+
+namespace IceInternal
+{
+
+ICE_API std::string timePointToString(
+    const std::chrono::system_clock::time_point&,
```

Comment:
The parameters should have names. It's also nice to include a comment that describes what the function does.